lib/fog/aws/models/compute/servers.rb

```diff
@@ -163,10 +163,13 @@ def save_many(template, min_servers = 1, max_servers = nil)
             # expect eventual consistency
             if (tags = server.tags) && tags.size > 0
               Fog.wait_for { server.reload rescue nil }
```

Comment:
I believe the original code was trying to use the wait_for on reload in the hopes that it would then be available for the create_tags. As you point out, this is insufficient, so we really want to wait_for the tag create.

That said, I think we could just drop this and only wait_for the create_tag now, what do you think?

Reply:
Sounds reasonable to me.  I didn't know if server.reload did something more so if you think it isn't needed then I can try it without it.

Reply:
Yeah, my read of it is that it was to protect the create_tags, not for some other side effect. But I could certainly be wrong, seems like it could be worth a shot to simplify and streamline though. Thanks!